App/views.py

```diff
@@ -0,0 +1,24 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+import json
+import requests
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the polls index.")    
+
+def main(request):
+	if(request.method=='POST'):
+		usr = request.POST['username']  #Here, we get the useranme fetched from html page
+		URL = "https://en.wikipedia.org/w/api.php?action=query&format=json&list=usercontribs&ucuser="
+		URL = URL + usr
+		r = requests.get(URL)  #Recieved data in json-format
+		lst = json.loads(r.text)
+		sub_lst = lst['query']['usercontribs'] #Contribution data extracted
+		try:
+			fetched_user = sub_lst[0]['user']
+			fetched_user_id = sub_lst[0]['userid']
+		except:
+			fetched_user = fetched_user_id = 'No user found'
```

Comment:
Or possibly the user was found but has no edits.

Reply:
okay , this was a loophole, will handle it :+1: